tests/basic_cxx_qt/CMakeLists.txt

```diff
@@ -16,9 +16,6 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Core Gui Test REQUIRED)
```

Comment:
Think these lines should still be there, as i know they come from the global root CMake scope, but it is just showing what the usecase might look like for the downstream applications. (and the other examples do have the find_package's)